java/src/main/java/TennisGame1.java

```diff
@@ -55,7 +51,11 @@ private String won(TennisPlayer winner) {
         return String.format("Win for %s", winner.getName());
     }
 
-    private boolean isNotPlaying(String playerName) {
-        return !Objects.equals(player1.getName(), playerName) && !Objects.equals(player2.getName(), playerName);
+    private void checkPlayerPresence(String playerName) {
+        Stream.of(player1, player2)
+                .map(TennisPlayer::getName)
```

Comment:
This is very familiar to me from Ruby. I had no idea this was possible.

Thank you.